org.es_de.emulationstation-de.yml

```
@@ -0,0 +1,82 @@
+#  SPDX-License-Identifier: MIT
+#
+#  EmulationStation Desktop Edition
+#  org.es_de.emulationstation-de.yml
+#
+#  Flatpak manifest file for use with flatpak-builder.
+#
+
+app-id: org.es_de.emulationstation-de
+sdk: org.freedesktop.Sdk
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+command: emulationstation
+branch: stable
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --device=all
+  - --filesystem=host:ro
+  - --filesystem=/var/lib/flatpak:ro
+  - --filesystem=home
+  - --filesystem=~/.var/app
+  - --share=network
+  - --talk-name=org.freedesktop.Flatpak
+  - --env=PATH=/app/bin:/usr/bin:/var/lib/flatpak/exports/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin:/var/lib/snapd/snap/bin
+cleanup:
+  - /include
+  - /share/ffmpeg
+  - /lib/cmake
+  - /lib/debug
+  - /lib/pkgconfig
+  - /lib/libfreeimage.a
+
+modules:
+  - name: ffmpeg
+    config-opts:
+    - --disable-static
+    - --disable-programs
+    - --disable-doc
+    - --enable-gpl
+    - --enable-shared
+    - --enable-libvorbis
+    - --enable-libopus
+    - --enable-libvpx
+    - --enable-postproc
+    sources:
+    - type: git
+      url: https://github.com/FFmpeg/FFmpeg.git
+      tag: n5.0
+
+  - name: freeimage
+    no-autogen: true
+    build-options:
+        cxxflags: -std=c++14
+    sources:
+    - type: archive
+      url: http://downloads.sourceforge.net/freeimage/FreeImage3180.zip
+      sha256: f41379682f9ada94ea7b34fe86bf9ee00935a3147be41b6569c9605a53e438fd
+    - type: shell
+      commands:
+      - sed -i 's|-o root -g root ||' ./Makefile.gnu
+      - sed -i 's|/usr|/app|' ./Makefile.gnu
+
+  - name: pugixml
+    buildsystem: cmake
+    config-opts:
+    - -DBUILD_SHARED_LIBS=on .
+    sources:
+    - type: archive
+      url: https://github.com/zeux/pugixml/releases/download/v1.11.4/pugixml-1.11.4.tar.gz
+      sha256: 8ddf57b65fb860416979a3f0640c2ad45ddddbbafa82508ef0a0af3ce7061716
+
+  - name: emulationstation
+    buildsystem: cmake
```

```suggestion
    buildsystem: cmake-ninja
```
unles it doesn't work

---

No problem, I'll change this.